pyvista/core/dataset.py

```diff
@@ -1198,12 +1213,17 @@ def scale(
             xyz = [xyz] * 3
 
         transform = _vtk.vtkTransform()
-        transform.Scale(xyz)
+        transform.Scale(xyz)  # type: ignore
         return self.transform(
             transform, transform_all_input_vectors=transform_all_input_vectors, inplace=inplace
         )
 
-    def flip_x(self, point=None, transform_all_input_vectors=False, inplace=False):
+    def flip_x(
+        self,
+        point: Optional[Vector] = None,
```

Comment:
methods earlier in the file with a `point` argument took FloatVector, here's it's just Vector

Reply:
```suggestion
        point: Optional[FloatVector] = None,
```